lib/records/P_RecDefs.h

```diff
@@ -100,21 +102,21 @@ struct RecConfigMeta {
 };
 
 struct RecRecord {
-  RecT rec_type;
-  const char *name;
-  RecDataT data_type;
-  RecData data;
-  RecData data_default;
+  RecT rec_type = RECT_NULL;
+  std::string name;
+  RecDataT data_type = RECD_NULL;
```

Comment:
Why not RecDataT data_type{RECD_NULL}; for consistency?

Reply:
Hmm are you sure they can't be, why?

https://godbolt.org/z/gSgfVn

Reply:
These particular ones cannot, at least with gcc 8 on F29. I changed it specifically due to a fail to compile.

Reply:
Hmmm works on Godbolt.  Even back to gcc 5.1.

https://godbolt.org/z/Q_Yvru

Reply:
Sadly, it doesn't work on our compilers in this case. You could try working with the actual code in situ, rather than simplified approximations somewhere else.